### PR TITLE
:memo: Document `Parser->parseWithoutContext`

### PR DESCRIPTION
**Pull request by homersimpsons**
Linked to https://github.com/symfony/symfony/pull/50144

### CODE REVIEW

File: components/expression_language.rst [hunk 1]
```diff
@@ -234,6 +234,14 @@ method after parsing any expression to get its AST::
     // dump the AST nodes as a string representation
     $astAsString = $ast->dump();
 
+.. note::
+
+    The :method:`Symfony\\Component\\ExpressionLanguage\\ExpressionLanguage::parse`
+    method requires you to be specific about the possible variables and functions.
+    If you do not need to specify them (e.g. for evaluating the AST in another
+    engine or to extract some insight from it) you can use the
+    :method:`Symfony\\Component\\ExpressionLanguage\\Parser::parseWithoutContext` method.
```
Comment (OskarStark) [inline components/expression_language.rst:243]:
```suggestion
    :method:`Symfony\\Component\\ExpressionLanguage\\Parser::parseWithoutContext` method.

        ..versionadded:: 6.4
        
            The `parseWithoutContext` method was introduced in Symfony 6.4.
```

Comment (homersimpsons) [reply components/expression_language.rst:243]: I am targeting symfony 6.3 should I target symfony 6.4 ?

Comment (OskarStark) [reply components/expression_language.rst:243]: Yes, but the branch is not yet created

**Comment by homersimpsons** [comment]
@OskarStark Do you know if I need to ping someone to review https://github.com/symfony/symfony/pull/50144 ?

~~(Will fix CI here later and I will rebase the other PR too)~~ => Done

**Comment by javiereguiluz** [comment]
Closing because the related code PR https://github.com/symfony/symfony/pull/50144 was closed in favor of https://github.com/symfony/symfony/pull/53806 which was documented in https://github.com/symfony/symfony-docs/pull/19529. Thanks!

**Pull request closed without merging**